portfoliostyle.css

```diff
@@ -0,0 +1,310 @@
+* {
+	Box-sizing: border-box;
+	font-family: monaco;
+	
+}
+
+div {
+	display: flex;
+}
+
+body {
+	/*background-color: #aaabb8;*/
+	background-image: url("images/snow-3161134_1920.jpg");
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+}
+
+nav {
+	display: flex;
+	background-color: #25274d;
+	color: #ffffff;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	padding: 0 25px 0 25px;
+	width: 100%;
+}
+
+nav button {
+	margin-left: 10px;
+	padding: 10px;
+	border-radius: 10px;
+	background-color: #29648a;
+	color: #ffffff;
+	border-color: #2e9cca;
+}
+
+main {
+	margin: 0 auto;
+	width: 90%;
+	flex-direction: column;
+	display: flex;
+	
+	font-size: 1.5vw;
+}
+
+
+.about {
+	display: flex;
+	flex-direction: column;
+	margin-top: 25px;
+	padding: 25px 25px 25px 25px;
+	background-color: #464866;
+	color: #ffffff;
+	border-radius: 10px;
+}
+
+.about div {
+	flex-direction: row
+	justify-content: space-between;
+}
+
+h2 {
+	margin-bottom: 0px;
+	margin-top: 0px;
+}
+.about img {
+	object-fit: scale-down;
+	margin-right: 25px;
+	max-width: 40%;
+}
+
+.about-projects {
+	display: flex;
+	flex-direction: column;
+}
+
+.projects {
+	display: flex;
+	flex-direction: column;
+	margin-top: 25px;
+	padding: 25px;
+	background-color: #464866;
+	color: #ffffff;
+	border-radius: 10px;
+}
+
+.project-container {
+	flex-direction: row;
+	justify-content: space-around;
+}
+
+.project-container div {
+	flex-direction: column;
+	align-items: center;
+}
+
+h3 {
+	margin-bottom: 0px;
+	margin-top: 0px;
+}
+
+.projects img {
+	width: 100px;
+	height: 100px;
+	margin: 10px 0 10px 0;
```

Comment:
Like with the padding on line 26 you can combine these and only give 2 values :)

Reply:
Done